doc/api/fs.markdown

```diff
@@ -577,7 +577,12 @@ If you want to be notified when the file was modified, not just accessed
 you need to compare `curr.mtime` and `prev.mtime`.
 
 _Note: when an `fs.watchFile` operation results in an `ENOENT` error, it will
-invoke the callback once. This is a change in functionality since v0.10._
+ invoke the listener once, with two stat objects, with all the fields set to
+ zero, except for the time fields, which are set to Thu Jan 01 1970 00:00:00.
```

Comment:
This is a run on sentence, that is a little bit confusing, with all the clauses. :)